packages/augur-core/source/contracts/reporting/RepPriceOracle.sol

```diff
@@ -1,25 +1,35 @@
-pragma solidity 0.5.10;
+pragma solidity 0.5.15;
 
 import 'ROOT/IAugur.sol';
 import 'ROOT/libraries/Initializable.sol';
 import 'ROOT/uniswap/interfaces/IUniswapV2.sol';
 import 'ROOT/uniswap/interfaces/IUniswapV2Factory.sol';
 import 'ROOT/reporting/IRepPriceOracle.sol';
+import 'ROOT/ens/IENSRegistry.sol';
+import 'ROOT/ens/IENSResolver.sol';
 
 
 contract RepPriceOracle is IRepPriceOracle, Initializable {
 
+    uint256 constant Q112 = 2**112;
+
+    bytes32 public constant UNISWAP_REGISTRY_ENS_NAME = 0xf259e1e59b6e1e9de21a18289a53332ebc255d2e4ece8a91e0e90c628a9c7f87; // "uniswapv2.eth"; // TODO: set this to the actual namehash of the owned ENS name
+
     IAugur public augur;
     address public cash;
-    uint256 public tau = 1 days / 15; // The time period (in blocks) of the twa, assuming a block is mined every 15 seconds. TODO: Review this value
+    uint256 public period = 3 days; // TODO: revisit if this is an appropriate period
```

Comment:
3 days seems too long for a simple moving average.  The longer it is, the more delayed the price feed will be and thus price changes won't be reflected quickly.  The shorter it is, the more at risk of manipulation attacks it is, though this is a function of market liquidity as well.

24 hours _feels_ good to me, but I don't have strong evidence to back that up.

Reply:
Unless we have something poking it when it isnt strictly needed we actually would need it to be 7 days. One idea is that Market creation _could_ poke this every time though that does add a fair bit of gas to it.